### PR TITLE
Compact run_examples

### PR DESCRIPTION
**Pull request by rgolangh**
Extract the repeating curl calls to a function so
reading this is bit easier.

Signed-off-by: Roy Golan <rgolan@redhat.com>

### CODE REVIEW

**Comment by rgolangh** [comment]
If the intention of the script is to be verbose explicitly to, say, copy paste examples and share them, then I'll close this PR

**Comment by eloycoto** [comment]
TBH, I prefer to keep verbose, as is the point where users can see/run without too much issue. But I'm happy to merge it

**Comment by rgolangh** [comment]
> TBH, I prefer to keep verbose, as is the point where users can see/run without too much issue. But I'm happy to merge it

but that is the point, I'm super ok with it verbose. ok to close by me. we can wait for one more review

**Review by pkliczewski — COMMENTED** [review]
/lgtm

File: workflow-examples/run_examples.sh [hunk 1]
```diff
@@ -112,14 +100,7 @@ wait_project_start() {
...
   exitOnFailure="$2"
-  response=$(curl -X 'POST' -s \
-    "${TARGET_URL}/api/v1/workflows" \
-    -H 'accept: */*' \
-    -H 'Authorization: Basic dGVzdDp0ZXN0' \
-    -H "X-XSRF-TOKEN: ${TOKEN}" \
-    -b $COOKIEFP \
-    -H 'Content-Type: application/json' \
-    -d "$params")
+  response=$(parodos_v1_curl workflows -d "$params")
```
Comment (nirarg) [inline workflow-examples/run_examples.sh:103]: Missing `-X 'POST'` here

Comment (rgolangh) [reply workflow-examples/run_examples.sh:103]: -d implies POST though. Infact I guess -X GET is also redundant most of the time here

**Comment by pkliczewski** [comment]
@rgolangh please rebase. I hope it will fix the failing test

**Pull request closed without merging**